app/models/tag_group.rb

```diff
@@ -3,4 +3,16 @@ class TagGroup < ActiveRecord::Base
   belongs_to :team
 
   validates :name, presence: true, length: { maximum: 15 }
+
+  def bg_color=(value)
```

Comment:
Missing test for this method.

Don't forget to test an invalid JSON too, because its behaviour.